FALSEHOODS.md

```diff
@@ -235,3 +235,10 @@ surprising how many false assumptions programmers continue to make about them.
     during, or after the actual implementation of numbering plan changes in the
     real world. The actual date on which a phone number range becomes active may
     not always adhere to official announcements.
+
+1.  **Phone numbers are numbers**
```

Comment:
Possible alternative titles:
"Phone numbers can be stored as numeric data types"
"Phone numbers can be treated as normal numbers"

Positioning:
I would put this after the "Users will only store phone numbers" one, since it is related in that if a user isn't storing a phone number then you *really* shouldn't treat it like a number.